webrtc-c/canary/src/Config.cpp

```diff
@@ -53,15 +101,14 @@ STATUS mustenvBool(CHAR const* pKey, Config::Value<BOOL>* pResult)
 STATUS optenvBool(CHAR const* pKey, Config::Value<BOOL>* pResult, BOOL defVal)
 {
     STATUS retStatus = STATUS_SUCCESS;
-    Config::Value<const CHAR*> raw;
+    Config::Value<Config::String> raw;
 
-    CHK_STATUS(optenv(pKey, &raw, NULL));
-    if (raw.value != NULL) {
-        if (STRCMPI(raw.value, "on") == 0 || STRCMPI(raw.value, "true") == 0) {
-            pResult->value = TRUE;
-        } else {
-            pResult->value = FALSE;
-        }
+    CHK(pResult != NULL, STATUS_NULL_ARG);
+    CHK(!pResult->initialized, retStatus);
+    CHK_STATUS(optenv(pKey, &raw, ""));
+
+    if (raw.value[0] != '\0') {
+        pResult->value = strtobool(raw.value);
```

Comment:
Nitpick: `strtobool ` -> `strToBool`

Reply:
This is intentional. It's consistent with other `strto*` APIs.